### PR TITLE
cross-*: use isl-devel and pkg-config

### PR DESCRIPTION
**Pull request by dkwo**
use isl-devel in the toolchain, as well as pkg-config, see #34153

[ci skip]

@leahneukirchen if this makes sense, I'll repeat it for the other cross-* packages

### CODE REVIEW

**Comment by leahneukirchen** [comment]
reasonable!

File: srcpkgs/cross-aarch64-linux-gnu/template [hunk 1]
```diff
@@ -6,11 +6,11 @@ _glibc_patchver="72-g0f90d6204d"
 _linux_version=5.10.4
 pkgname=cross-aarch64-linux-gnu
 version=0.35
-revision=3
+revision=4
 build_style=void-cross
 configure_args="--with-arch=armv8-a"
 hostmakedepends="texinfo tar gcc-objc gcc-go flex perl python3"
-makedepends="isl15-devel libmpc-devel gmp-devel mpfr-devel
+makedepends="isl-devel libmpc-devel pkg-config gmp-devel mpfr-devel
```
Comment (oreo639) [inline srcpkgs/cross-aarch64-linux-gnu/template:13]: pkg-config should be in hostmakedepends.

Comment (dkwo) [reply srcpkgs/cross-aarch64-linux-gnu/template:13]: thanks

**Comment by dkwo** [comment]
will do the others soon :)

**Comment by dkwo** [comment]
Done!

**Comment by dkwo** [comment]
just as a ref to self:
```
for a in cross-aarch64-linux-gnu cross-aarch64-linux-musl cross-arm-linux-gnueabi cross-arm-linux-gnueabihf cross-arm-linux-musleabi cross-arm-linux-musleabihf cross-armv7l-linux-gnueabihf cross-armv7l-linux-musleabihf cross-i686-linux-musl cross-i686-pc-linux-gnu cross-mipsel-linux-musl cross-mipsel-linux-muslhf cross-mips-linux-musl cross-mips-linux-muslhf cross-powerpc64le-linux-gnu cross-powerpc64le-linux-musl cross-powerpc64-linux-gnu cross-powerpc64-linux-musl cross-powerpcle-linux-gnu cross-powerpcle-linux-musl cross-powerpc-linux-gnu cross-powerpc-linux-musl cross-x86_64-linux-gnu cross-x86_64-linux-musl; do
	sed -i s/isl15/isl/ srcpkgs/$a/template
	sed -i 's/python3/python3 pkg-config/' srcpkgs/$a/template
	xrevbump "makedepends on isl-devel pkg-config" $a
done
```

**Comment by leahneukirchen** [comment]
Let's churn this on the weekend then.

**Comment by oreo639** [comment]
I don't have an issue with this PR the changes could probably pull the musl changes from: https://github.com/void-linux/void-packages/pull/42931

**Comment by dkwo** [comment]
@oreo639 I have not tested that patch though, as I'm on musl 1.2

**Comment by dkwo** [comment]
@leahneukirchen Can this be merged, either with or without the musl patch?

**Comment by leahneukirchen** [comment]
Yes, but it needs a rebase...

**Comment by dkwo** [comment]
Done!

**Comment by dkwo** [comment]
Rebased. @leahneukirchen Can you take a look?

**Comment by leahneukirchen** [comment]
thanks for your patience!

**Pull request merged**